Source/JavaScriptCore/API/JSContextRefPrivate.h

```diff
@@ -101,15 +101,15 @@ JS_EXPORT void JSContextGroupClearExecutionTimeLimit(JSContextGroupRef group) JS
 @result The value of the setting, true if remote inspection is enabled, otherwise false.
 @discussion Remote inspection is true by default.
 */
-JS_EXPORT bool JSGlobalContextGetRemoteInspectionEnabled(JSGlobalContextRef ctx) JSC_API_AVAILABLE(macos(10.10), ios(8.0));
+JS_EXPORT bool JSGlobalContextGetRemoteInspectionEnabled(JSGlobalContextRef ctx) JSC_API_AVAILABLE(macos(10.10), ios(8.0)) JSC_API_DEPRECATED_WITH_REPLACEMENT("JSGlobalContextIsInspectable", macos(10.10, JSC_MAC_TBA), ios(8.0, JSC_IOS_TBA));
```

Comment:
I don't think we keep the `API_AVAILABLE` around when we add a `API_DEPRECATED_WITH_REPLACEMENT`